### PR TITLE
[Outlook] (Smart Alerts) Add a recommendation note and on-send comparison table

### PR DESCRIPTION
**Pull request by samantharamon**
Related reference PR: https://github.com/OfficeDev/office-js-docs-reference/pull/1428

### CODE REVIEW

**Review by Rick-Kirkham — CHANGES_REQUESTED** [review]
This is going to cause merge conflicts with https://github.com/OfficeDev/office-js-docs-pr/pull/3682/files, which I can't merge for another couple of days. Also, much of this content will need to be updated for the JSON manifest. Can we hold off on this one until it can be merged smoothly?

**Comment by samantharamon** [comment]
> This is going to cause merge conflicts with https://github.com/OfficeDev/office-js-docs-pr/pull/3682/files, which I can't merge for another couple of days. Also, much of this content will need to be updated for the JSON manifest. Can we hold off on this one until it can be merged smoothly?

No worries. I was just about to leave a comment in your PR actually. :) I'll keep an eye out for your merged changes. I already have a revised copy of the Smart Alerts and on-send comparison table, which includes JSON manifest notes. I'll integrate it into this PR once your work is complete and request for your review.

File: docs/outlook/outlook-on-send-addins.md
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/outlook-on-send-addins.md:20]:
```suggestion
> [Smart Alerts](smart-alerts-onmessagesend-walkthrough.md) includes a newer version of the on-send feature. It was released in [requirement set 1.12](/javascript/api/requirement-sets/outlook/requirement-set-1.12/outlook-requirement-set-1.12) and introduced the `OnMessageSend` and `OnAppointmentSend` events. Similar to the on-send feature, Smart Alerts enables your add-in to check that certain conditions are met before a mail item is sent. Smart Alerts differentiates itself from the on-send feature as follows:
```

File: docs/outlook/outlook-on-send-addins.md
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/outlook-on-send-addins.md:17]: In the AppSource bullet of the Limitations section, I think it would be helpful to include a link to the comparison table or to the store options article. Maybe append something like: "However, Smart Alerts is a newer feature which can provide AppSource support for similar functionality. To learn more, see *{some link}*."

Comment (samantharamon) [reply docs/outlook/outlook-on-send-addins.md:17]: Thanks for the suggestion. Links have been added. I linked to the overall Smart Alerts walkthrough, instead of the comparison table, so that the user gets an introduction to the feature before diving into specific details.

**Comment by Rick-Kirkham** [comment]
> > This is going to cause merge conflicts with https://github.com/OfficeDev/office-js-docs-pr/pull/3682/files, which I can't merge for another couple of days. Also, much of this content will need to be updated for the JSON manifest. Can we hold off on this one until it can be merged smoothly?
> 
> @Rick-Kirkham No worries. I was just about to leave a comment in your PR actually. :) I'll keep an eye out for your merged changes. I already have a revised copy of the Smart Alerts and on-send comparison table, which includes JSON manifest notes. I'll integrate it into this PR once your work is complete and request for your review.

I have just merged https://github.com/OfficeDev/office-js-docs-pr/pull/3682

**Comment by samantharamon** [comment]
> > > This is going to cause merge conflicts with https://github.com/OfficeDev/office-js-docs-pr/pull/3682/files, which I can't merge for another couple of days. Also, much of this content will need to be updated for the JSON manifest. Can we hold off on this one until it can be merged smoothly?
> > 
> > 
> > @Rick-Kirkham No worries. I was just about to leave a comment in your PR actually. :) I'll keep an eye out for your merged changes. I already have a revised copy of the Smart Alerts and on-send comparison table, which includes JSON manifest notes. I'll integrate it into this PR once your work is complete and request for your review.
> 
> I have just merged #3682

Thanks, @Rick-Kirkham. I added the Teams manifest information to the comparison table for your review.

**Review by Rick-Kirkham — APPROVED** [review]

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**
